src/main/java/net/masterthought/cucumber/json/Match.java

```diff
@@ -1,13 +1,95 @@
 package net.masterthought.cucumber.json;
 
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.LinkedList;
+import java.util.List;
+
 public class Match {
 
     // Start: attributes from JSON file report
-    private final String location = null;
+    private final JsonElement location = null;
     // End: attributes from JSON file report
 
     public String getLocation() {
```

Comment:
Hi, I built the plugin again using these steps:

git clone -b  location https://github.com/tommywo/cucumber-reporting.git
git clone https://github.com/jenkinsci/cucumber-reports-plugin.git
cd cucumber-reporting
mvn install
cd ../cucumber-reports-plugin
mvn versions:set -DartifactId=cucumber-reporting
-DgroupId=net.masterthought -DoldVersion=1.3.0
-DnewVersion=1.3.0-SNAPSHOT
mvn package

And got the same error


Thanks,

Sergiu


On 7 March 2016 at 19:33, Damian Szczepanik <notifications@github.com>
wrote:

> In src/main/java/net/masterthought/cucumber/json/Match.java
> <https://github.com/damianszczepanik/cucumber-reporting/pull/316#discussion_r55257960>
> :
>
> >  public class Match {
> >
> >      // Start: attributes from JSON file report
> > -    private final String location = null;
> > +    private final JsonElement location = null;
> >      // End: attributes from JSON file report
> >
> >      public String getLocation() {
>
> [image: Codacy]
> <https://camo.githubusercontent.com/37ddd55708a38212f4e9de4e3eac98358fefe597/68747470733a2f2f7777772e636f646163792e636f6d2f6173736574732f696d616765732f66617669636f6e2e706e67>
> Issue found: The method getLocation() has an NPath complexity of 298
> <https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2183564530/issues/source?bid=3059308&fileBranchId=3097149#l17>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/damianszczepanik/cucumber-reporting/pull/316/files#r55257960>
> .
>